src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -2782,9 +2782,9 @@ namespace Microsoft.FSharp.Core
         /// <example id="right-shift-example-1">
         /// <code lang="fsharp">
         /// let a = 206      // 00000000000000000000000011010000
-        /// let c1 = a &gt;&gt; 2  // 00000000000000000000000000110100
+        /// let c1 = a &gt;&gt;&gt; 2  // 00000000000000000000000000110100
         /// // Evaluates to 51
-        /// let c2 = a &gt;&gt; 6  // 00000000000000000000000000000011
+        /// let c2 = a &gt;&gt;&gt; 6  // 00000000000000000000000000000011
```

Comment:
Do we not reformat '>' in comments to '&gt;' when we generate xml doc comments.  It seems like we absolutely should if we don't currently.  Because having to format your doc comment as xml compatible code seems like unnecessary pain.

Reply:
My understanding is that C# needs escapes too within XML docs, see https://stackoverflow.com/questions/7387529/how-to-show-char-in-c-sharp-xml-comments

I assume this applies in `<code ..>...` sections too

Reply:
Though looking at https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/examples there are code samples that are unescaped (using only `>` though, not `<`)

Reply:
Actually F# is the same as C# here - you can use `>` but can't use `<`, and the same XML is produced

```
/// <summary>
///   a > b 
/// </summary>
```